### ISSUE TITLE
Automotive Play buttons

### ISSUE THREAD

**Issue by beth-edenspiekermann** [android/uamp#533]
When I run the automotive module in the Polestar 2 or Landscape emulator, the play button is not responsive. I can start playing a song, but I have to open the settings app and force quit the app to stop it.

Somebody should fix that!

**Comment by vishtech36** [comment]
The play/pause button consistently displays the play triangle icon at all times, and this issue has been reported by numerous users with the latest 2.7 version. Notably, users have encountered the same problem with the Spotify app. You can find further details in the following link: [problems with the play/pause button in 2.7](https://www.reddit.com/r/Polestar/comments/11u4e8n/is_anybody_else_having_problems_with_the/). It's important to note that this issue is not related to the code but is likely attributed to a firmware update. I offer my support in addressing this matter.

**Comment by beth-edenspiekermann** [comment]
@vishtech36 The team for this open source sample is at Google with access to the source code for Automotive/Google Assistant Auto, so, how is this issue not related to the code? Maybe there is a workaround. It's not simply the display of the button - it's the ability to play and stop playback.

Since there is no way to stop the app playing, this should be defined as a problem with the highest P1 severity.

**Comment by vishtech36** [comment]
@beth-edenspiekermann  I totally agree with you, but it seems it's only affecting Polestar, and it appears to be a system bug within that OS. From my experience, it seems like there's an internal bug at the OS level that isn't triggering a specific event to prompt the app to change its icon or perform the intended action. Additionally, Polestar is customizing the core AOSP, which is beyond our control.

**Comment by beth-edenspiekermann** [comment]
Is the Google team maintaining this example @vishtech36? If so, they have access to the Automotive AOSP.
Isn't it on both Polestar and the landscape emulator? I tested 2 different emulators.
